lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -29,6 +29,9 @@ function dosomething_campaign_preprocess_node(&$vars) {
   // Add inline css based on vars.
   dosomething_helpers_add_inline_css($vars);
 
+  // Add internal js script based on vars.
+  dosomething_helpers_add_js($vars);
```

Comment:
move adding onboaring JS down below the checks for pitch page/sms and closed campaigns